src/v2/Apps/Artwork/Components/ArtworkSidebar/__tests__/ArtworkSidebarAuctionProgressBar.jest.tsx

```diff
@@ -0,0 +1,69 @@
+import { ArtworkSidebarAuctionProgressBar_Test_QueryRawResponse } from "v2/__generated__/ArtworkSidebarAuctionProgressBar_Test_Query.graphql"
+import { renderRelayTree } from "v2/DevTools"
+import { graphql } from "react-relay"
+import { ArtworkSidebarAuctionProgressBarFragmentContainer } from "../ArtworkSidebarAuctionProgressBar"
+import { DateTime } from "luxon"
+
+jest.unmock("react-relay")
+
+describe("ArtworkSidebarAuctionProgressBar", () => {
+  const getWrapper = async (
+    response: ArtworkSidebarAuctionProgressBar_Test_QueryRawResponse["artwork"]
+  ) => {
+    return renderRelayTree({
```

Comment:
@mzikherman - we've been using automocking relay test utils for years now 😉 Please use that instead of our homegrown test tools.

Reply:
<img width="238" alt="Screen Shot 2022-04-27 at 10 46 37 AM" src="https://user-images.githubusercontent.com/236943/165587983-faba11aa-4667-465b-a716-3e31bde28120.png">

Reply:
@damassi could that be enforced via auto-complete or linting or automation somewhere? Otherwise it requires a dev 'in the know' to happen to see it!

Reply:
This does also use the same test style as the other specs in the artwork app, so it's consistent in that way. Additionally, it's going to be shortly moved into another component (this was more of a placeholder) which uses the old style specs itself.

In general it can be hard to keep track of what the currently accepted approaches are (hooks vs. classes, various testing approaches), esp. as we don't go back and always update older versions to the new one. Maybe we should have a playbook for these that is kept up to date? (I'm curious your recommendations to get better about that, esp. as an infrequent Force contributor).

Reply:
Even if we don't rely on automation,  a note in [this doc](https://github.com/artsy/force/blob/main/docs/best_practices.md) might be a good idea.

Reply:
@mzikherman - always look to our best practices doc! We have a section on testing in there: https://github.com/artsy/force/blob/main/docs/best_practices.md#write-unit-tests-for-new-components

Reply:
And even if the rest of the app is using old tooling, all new code / files should use relay auto mocking so as to not create more tech debt.

Reply:
Oh, I had overlooked that doc! 😍  That looks great.